### PR TITLE
added post Start-Up Deep Dive  : Founder Requirements & Product Ideation

### PR DESCRIPTION
**Pull request by hacksh4w**

### CODE REVIEW

File: content/post/session1-startupdeepdive.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+<br>
+<img src="/images/session1-startupdeepdive/image1.jpg" width="550" height="auto">
+<br> 
+
+The speaker of the session, Mr. Haroon Shahul talked about his experiences in MEC as well as the origin story of Solwearth Ecotech Pvt. Ltd. He primarily focused on infusing technology and our social responsibility to bring about sustainable development products and practices. He shared his career experiences as a CEO, COO, GCO, and Board member in the telecom and IT industries in Asia, the Middle East, and North Africa and stories of the companies he developed from scratch into sizable businesses in Asia, Middle East, and Northern African markets. Explained the whole ideation process of Solwearth from scratch. The idea came from a friend's reunion on the discussion on Social responsibility as an engineer. The development of a product from an idea to a fully finished version, and finding the right people to build the product, this process gives us an idea about what things need to be considered when looking for a co-founder. When starting a new business, it can be helpful to have a co-founder or team to work with.
```
Comment (aldrinjenson) [inline content/post/session1-startupdeepdive.md]: Can we make the speaker name bolded please

File: content/post/session1-startupdeepdive.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Start-up Deep Dive : Founder Requirements and Product Ideation"
+date: 2023-01-17T06:21:55+05:30
+authors:
+  - nikesh-t-v
+Description: "A participant’s take away from the first session of Startup Deep Dive, taken by Mr. Haroon Shahul on the topic \"Founder Requirements\"."
```
Comment (aldrinjenson) [inline content/post/session1-startupdeepdive.md]: Description: A participant’s take away from the first session of Startup Deep Dive, taken by Mr. Haroon Shahul on the topic "Founder Requirements"

Can you please test if removing the parent quotes altogether works fine? Or if we could use 'single quotes' for teh parent wrapper.. If so we can avoid the escape characters and that's altogether better no?

File: content/post/session1-startupdeepdive.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Start-up Deep Dive : Founder Requirements and Product Ideation"
+date: 2023-01-17T06:21:55+05:30
+authors:
+  - nikesh-t-v
+Description: "A participant’s take away from the first session of Startup Deep Dive, taken by Mr. Haroon Shahul on the topic \"Founder Requirements\"."
+Tags: ["startup","startupdeepdive","founder-requirements"]
```
Comment (aldrinjenson) [inline content/post/session1-startupdeepdive.md]: Please add "ets" as a tag for easy filtering later

**Comment by aldrinjenson** [comment]
Hey @hacksh4w this looks great! Appreciate the efforts✨🎉

Please remove the extra image(image4/5) added and I think we're good to go.

File: content/post/session1-startupdeepdive.md (hunk 2 above)
Comment (hacksh4w) [reply content/post/session1-startupdeepdive.md]: removing the parent quotes does work fine! Nice catch

File: content/post/session1-startupdeepdive.md (hunk 1 above)
Comment (hacksh4w) [reply content/post/session1-startupdeepdive.md]: Yeah sure, 
Done!

File: content/post/session1-startupdeepdive.md (hunk 2 above)
Comment (aldrinjenson) [reply content/post/session1-startupdeepdive.md]: Settt. Closing this.

**Pull request merged**